src/main/java/org/commcare/suite/model/DetailField.java

```diff
@@ -49,6 +49,8 @@ public class DetailField implements Externalizable {
     @Nullable
     private EndpointAction endpointAction;
 
+    private boolean showBorder;
```

Comment:
Feels like this should be part of [Style](https://github.com/dimagi/commcare-core/blob/master/src/main/java/org/commcare/suite/model/Style.java) instead